pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1,2240 +1,2283 @@
-﻿/*
```

Comment:
This one looks like just a line endings change. I'll see if I can clean that up.

Reply:
...but of course when I compare it to master, WinMerge doesn't show any line ending or other whitespace change, There are a handful of actual changes, so I'll try to sort this out.